apps/frontend/src/polyfills.ts

```diff
@@ -1,7 +0,0 @@
-/**
```

Comment:
Does this mean we aren't using polyfills? or it's just used in a different way?

Reply:
When i generated a new app, and look at their config, they dont include the polyfill anymore, so either its deprecated, or... in theory webpack does the poly fill already, and this is just a override that we are not using